### PR TITLE
#5548 empty state

### LINKED ISSUE #5548: Empty state for collection activity
- we are currently missing a message if there is no activity/filtered activity for the collection
image
- so if you for example filter sale and there is none, it will display a message

"Looks like there's nothing here yet! 
Check back later or reset your filters
image

first sentence has 20px font size, the other is just regular text

### PR DESCRIPTION
**Pull request by prachi00**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

## Context

- [x] Closes #5548
- [ ] Requires deployment <>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [ ] I've tested it at </bsx/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [x] Fill up your KSM address: 
[Payout](https://beta.kodadot.xyz/transfer/?target=EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.
Screen Shot 2023-04-08 at 5 42 05 PM

### CODE REVIEW

**Comment by daiagi** [comment]
add some bottom padding on mobile?

image

**Comment by exezbcz** [comment]
~add some bottom margin on mobile please~

haha, you were faster :D 

maybe thinking about making the text bigger - in the issue is - heading 20px - we can do bold 
subheading - 16px font weight normal


Otherwise oki, thanks!

**Comment by prachi00** [comment]
> ~add some bottom margin on mobile please~
> 
> haha, you were faster :D
> 
> maybe thinking about making the text bigger - in the issue is - heading 20px - we can do bold subheading - 16px font weight normal
> 
> Otherwise oki, thanks!
Screen Shot 2023-04-09 at 5 43 01 PM

done

File: components/collection/activity/events/Events.vue [hunk 1]
```diff
@@ -21,6 +21,14 @@
           <span>{{ $t('activity.event.time') }}</span>
         </div>
       </div>
+      <div
+        v-if="!displayedEvents.length"
+        class="is-flex is-flex-direction-column is-align-items-center is-justify-content-center pt-6 pb-6">
```
Comment (daiagi) [inline components/collection/activity/events/Events.vue:26]:
```suggestion
        class="is-flex is-flex-direction-column is-align-items-center is-justify-content-center py-6">
```

**Comment by exezbcz** [comment]
one small thing, text align center please
image

**Review by vikiival — COMMENTED** [review]
Looks oki

**Comment by vikiival** [comment]
pay 10 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $10 @ 33.49 USD/KSM ~ 0.299 $KSM
🧗 [EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU](https://kusama.subscan.io/account/EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)
🔗 [0xb0a2deaf7b636f92dc3976fd9a6ff23acc6ca0d4135af79c52a1e1c8d47af5b7](https://kusama.subscan.io/extrinsic/0xb0a2deaf7b636f92dc3976fd9a6ff23acc6ca0d4135af79c52a1e1c8d47af5b7)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**